tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -0,0 +1,56 @@
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: For openSUSE virtualization test only. login in console, install kvm/xen patterns if needed.
+#          Even if you'd like to run tests without host installation(IPMI_DO_NOT_RESTART_HOST=1), this module is still necessary as login console in this module is required.
+# Maintainer: Julie CAO <jcao@suse.com>
+
+use strict;
+use warnings;
+use base 'virt_autotest_base';
+use testapi;
+use ipmi_backend_utils;
+use Utils::Backends qw(use_ssh_serial_console is_remote_backend);
+use utils qw(zypper_call systemctl);
+use virt_autotest::utils qw(is_kvm_host is_xen_host);
+
+sub run {
+
+    #skip TW host installation and directly login
+    if (get_var('IPMI_DO_NOT_RESTART_HOST')) {
+        select_console 'sol', await_console => 0;
+        send_key 'ret';
+    }
+
+    use_ssh_serial_console unless check_screen('text-logged-in-root');
+
+    return if get_var('IPMI_DO_NOT_RESTART_HOST');
```

Comment:
Why return after   select_console 'sol', await_console => 0; and use_ssh_serial_console ?

Reply:
We assume the host is ready(include kvm and xen patterns) when IPMI_DO_NOT_RESTART_HOST is used.